apps/docs/content/components/River/index.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: River
+description: Use the river component to introduce a feature using a type and media pairing.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![A river component used to showcase a security feature. The title reads "Get security feedback with every git push" and it includes an image showing screenshots of of a CLI prompt with the git push command and a code scanning screenshot.](https://github.com/primer/brand/assets/175638/203f49ae-e87c-4b66-8df0-22a1c8bad1b6)
+
+## Usage
+
+Use rivers to showcase features and introduce topics. Rivers compose images or videos and text together. River's text includes a heading, body text, and an optional link.
+
+River content should be short and concise, no longer than 3 or 4 sentences and in a single paragraph when possible.
+
+<DoDontContainer>
+  <Do>
+    <img
+      alt=""
+      src="https://github.com/primer/brand/assets/175638/8b3057f9-00c5-4bf5-af44-00b0129c6c6e"
+    />
+    <Caption>Write short and concise text content.</Caption>
+  </Do>
+  <Dont>
+    <img
+      alt=""
+      src="https://github.com/primer/brand/assets/175638/ea902be3-2032-49de-bdb4-23be0c0f090d"
+    />
+    <Caption>Don't overload the river with too much content.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Stacked
+
+Two or more rivers can be stacked to walk the user through a set of feautres. When stacking rivers, consider alternating between left and right alignments to create a more dynamic reading flow. When alternating alignements, use a `40:60` image to text ratio, when not, keep the ratio to `50:50`.
```

Comment:
```suggestion
Two or more rivers can be stacked to guide the user through a set of feautres. When stacking rivers, use alternating between left and right alignments to create a more dynamic reading flow. When alternating alignements, use a `40:60` image to text ratio, otherwise keep the ratio to `50:50`.
```

Reply:
in case you don't use this suggestion, there's also a typo that should be fixed ☝️